settings/src/client.rs

```diff
@@ -123,10 +105,8 @@ fn default_balance_notification() -> bool {
 /// to a exit and to setup the exit tunnel
 #[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
 pub struct ExitClientSettings {
-    #[serde(rename = "exits", default)]
-    pub old_exits: HashMap<String, OldExitServer>,
     /// This stores a mapping between an identifier (any string) to exits
-    #[serde(rename = "new_exits", default)]
+    #[serde(default)]
```

Comment:
since we have routers in the field with `old_exits` as the name for this field in the config we'll need to use the serde [alias](https://serde.rs/field-attrs.html#alias) attribute to rename it. So put `#[serde(alias = "new_exits", default)]` here.